cg/apps/slurm/sbatch.py

```diff
@@ -25,7 +25,8 @@
 #SBATCH --job-name={job_name}
 #SBATCH --partition=dragen
 #SBATCH --account={account}
-#SBATCH --cpus-per-task=24
+#SBATCH --cpus-per-task=48
+#SBATCH --exclusive
```

Comment:
I think this alone should result in the change we want
```suggestion
#SBATCH --cpus-per-task={cpus_per_task}
```

Reply:
I think adding --exclusive is redundant since each node has 48 cores.